NDTensors/test/Project.toml

```diff
@@ -1,4 +1,5 @@
 [deps]
+AMDGPU = "21141c5a-9bdb-4563-92ae-f87d6854732e"
```

Comment:
Reminder to move this to `Extras` for now with `Metal` while we determine how to test .

Reply:
Moved for now.